build.gradle

```
@@ -786,7 +786,7 @@ dependencies {
         java17Dependencies("com.github.GTNewHorizons:lwjgl3ify:${lwjgl3ifyVersion}")
     }
     if (modId != 'hodgepodge') {
-        java17Dependencies('com.github.GTNewHorizons:Hodgepodge:2.4.4')
+        java17Dependencies('com.github.GTNewHorizons:Hodgepodge:2.3.35')
```

keep this pinned at 2.4.4 -- I need to PR that upstream
```suggestion
        java17Dependencies('com.github.GTNewHorizons:Hodgepodge:2.4.4')
```